fieldservice_repair/models/fsm_order.py

```diff
@@ -12,32 +12,32 @@ class FSMOrder(models.Model):
     repair_id = fields.Many2one(
         'repair.order', 'Repair Order')
 
-    @api.model
-    def create(self, vals):
-        # if FSM order with type repair is created then
-        # create a repair order
-        order = super(FSMOrder, self).create(vals)
-        if order.type == 'repair':
-            if (order.equipment_id and
-                    order.equipment_id.current_stock_location_id):
-                equipment = order.equipment_id
-                repair_id = self.env['repair.order'].create({
-                    'name': order.name or '',
-                    'product_id': equipment.product_id.id or False,
-                    'product_uom': equipment.product_id.uom_id.id or False,
-                    'location_id':
-                        equipment.current_stock_location_id and
-                        equipment.current_stock_location_id.id or False,
-                    'lot_id': equipment.lot_id.id or '',
-                    'product_qty': 1,
-                    'invoice_method': 'none',
-                    'internal_notes': order.description,
-                    'partner_id':
-                        order.customer_id and order.customer_id.id or False,
-                })
-                order.repair_id = repair_id
-            elif not order.equipment_id.current_stock_location_id:
-                raise ValidationError(_("Cannot create Repair Order because "
-                                        "Equipment does not have a Current "
-                                        "Inventory Location."))
-        return order
+    # @api.model
```

Comment:
Hi @osi-scampbell, I tested the fieldservice_repair module, but I saw that the CREATE method was commented. Do you have more information on why? Do you see any problem with uncommenting this method?